server/service/src/sensor/update.rs

```diff
@@ -49,28 +49,31 @@ pub fn validate(
         Some(sensor_row) => sensor_row,
         None => return Err(UpdateSensorError::SensorDoesNotExist),
     };
-
-    if !check_sensor_serial_is_unique(&input.id, input.serial.clone(), connection)? {
-        return Err(UpdateSensorError::SerialAlreadyExists);
-    }
-
```

Comment:
Out of interest, why was this check removed?

Reply:
We're not doing mutations for these fields, can be added back in when necessary